02-getting-account-data/index.js

```diff
@@ -0,0 +1,39 @@
+// SDK exposes all you need
+import { Asset, Chains, Int64, UInt64 } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account representation (optional 2nd parameter for chain, could use a default if not specified)
+    const account: Account = Account.from('teamgreymass', Chains.EOS)
+
+    // Retrieves data from the APIs (v1/chain/get_account)
+    await account.loadData()
```

Comment:
Continuing from the other thread - the above takes the `Account` and makes it a representation of an account, and then has a `loadData` method to make the API call and populate the data. 

I'm not sure if we want to use this pattern. It has it's pros/cons.

- Pro: Allows you to just call `loadData` to load/refresh the `Account` data. Could have a timestamp to indicate some form of staleness.
- Con: Always feels weird to just await an async function like this to me.

Reply:
Actually lets continue this discussion in a new PR. I'm going to merge this. 

If anyone has suggestions on how this best changes, open a PR!x